src/query/frame_change.hpp

```diff
@@ -70,7 +69,7 @@ struct CachedValue {
   }
 
  private:
-  bool IsValueInVec(const std::vector<TypedValue> &vec_values, const TypedValue &value) const {
+  static bool IsValueInVec(const std::vector<TypedValue> &vec_values, const TypedValue &value) {
```

Comment:
Why did you drop `const`?

Reply:
It doesn't make sense that static method is const